<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>News Articles</title>
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="icon" type="image/png" href="/images/logo/Logo_Round_32x32.png">
    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&display=swap" rel="stylesheet">
    <link href="/assets/css/style.css" rel="stylesheet">
    
    <meta property="og:title" content="Partnership to Advance Throughput Computing" />
    <meta property="og:type" content="website" />
    <meta property="og:url" content="https://path-cc.io" />
    <meta property="og:image" content="https://path-cc.io/images/logo/Logo_Primary_Acronym.png" />
    <meta property="og:description" content="PATh brings together the Center for High Throughput Computing and the Open Science Grid in order to advance the nation’s campuses and science communities through the use of distributed High Throughput Computing." />
    <meta name="twitter:card" content="summary" />
    <meta name="twitter:site" content="PATh" />
    <meta name="twitter:creator" content="" />
    <meta name="twitter:image" content="images/logo/Logo_Round_Med.png" />

    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
</head>
<body class='page '>
<div id="main-menu-mobile" class="main-menu-mobile">
  
  <ul>
    
    <li class="">
      
      <a href="/facility/">Facility</a>
    </li>
    
    <li class="">
      
      <a href="/services/">Services</a>
    </li>
    
    <li class="active">
      
      <a href="/news/">News</a>
    </li>
    
    <li class="">
      
      <a href="/events/">Events</a>
    </li>
    
    <li class="">
      
      <a href="/contact/">Contact</a>
    </li>
    
  </ul>
</div>
<div id="wrapper" class="wrapper">
    



<div>
  <div class="container-lg">
    <nav class="navbar navbar-expand-lg navbar-light ps-0 pe-0">

      <a class="navbar-brand" href="/">
        <img height="70px" src="/images/logo/Logo_Round_Med.png" alt="PATh"/>
      </a>

      <div id="navbarNavDropdown" class="collapse navbar-collapse justify-content-between">
        <ul class="navbar-nav">
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="#" id="about-nav" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
              About
            </a>
            <div class="dropdown-menu" aria-labelledby="about-nav">
              <a class="dropdown-item" href="/about">PATh</a>
              <a class="dropdown-item" href="/team">Team</a>
              <a class="dropdown-item" href="/glossary">Glossary</a>
            </div>
          </li>
          
          
          <li class="nav-item ">
            <a class="nav-link" href="/facility/">Facility</a>
          </li>
          
          <li class="nav-item ">
            <a class="nav-link" href="/services/">Services</a>
          </li>
          
          <li class="nav-item active">
            <a class="nav-link" href="/news/">News</a>
          </li>
          
          <li class="nav-item ">
            <a class="nav-link" href="/events/">Events</a>
          </li>
          
          <li class="nav-item ">
            <a class="nav-link" href="/contact/">Contact</a>
          </li>
          
        </ul>
        <div class="social-media">
          <a class="text-decoration-none" href="https://twitter.com/opensciencegrid">
            <img alt="Twitter Icon" src="/images/icons/twitter.svg"/>
          </a>
          <a class="text-decoration-none" href="https://www.youtube.com/channel/UCd1UBXmZIgB4p85t2tu-gLw">
            <img src="/images/icons/youtube.svg"/>
          </a>
        </div>
      </div>

      <button id="toggle-main-menu-mobile" class="hamburger hamburger--slider" type="button" aria-label="Mobile Menu">
  <span class="hamburger-box">
    <span class="hamburger-inner"></span>
  </span>
</button>
    </nav>
  </div>
</div>

    <main>
        <div class="container pb-5">
            <div class="pt-5">
    <h1 class="title">News Articles
    
</h1>

</div>
<div class="row">
  
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-03-19-Nicholas-Cooley-2021-David-Swanson-Award-Winner/">Connor Natzke awarded David Swanson Memorial Award</a>
          
          <small class="text-muted article-date">April 11, 2022</small>
          
        </h2>
        <p>Connor Natzke was awarded the 2022 David Swanson Memorial Award at the March OSG All-Hands Meeting. The memorial was established to honor our late colleage David Swanson who contributed to campus research across the country.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2022-04-11-OSGAHM2022-Summary/">Celebrating a dynamic OSG All-Hands Meeting 2022</a>
          
          <small class="text-muted article-date">April 11, 2022</small>
          
        </h2>
        <p>In March, 251 OSG users, resource providers, and staff convened virtually for the OSG All-Hands Meeting 2022. This article provides a brief summary of the talks and discussions that took place, and includes links to the video recordings of all talks.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-12-16-precision-mental-health/">Harnessing HTC-enabled precision mental health to capture the complexity of smoking cessation</a>
          
          <small class="text-muted article-date">December 16, 2021</small>
          
        </h2>
        <p>Collaborating with CHTC research computing facilitation staff, UW-Madison researcher Gaylen Fronk is using HTC to improve cigarette cessation treatments by accounting for the complex differences among patients.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-11-09-usgs-invasivespecies/">Protecting ecosystems with HTC</a>
          
          <small class="text-muted article-date">November 09, 2021</small>
          
        </h2>
        <p>Researchers at the USGS are using HTC to pinpoint potential invasive species for the United States.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-10-26-bat-genomics/">80,000 jobs, 40 billion base pairs, and 20 bats –– all in 4 weeks</a>
          
          <small class="text-muted article-date">October 26, 2021</small>
          
        </h2>
        <p>An evolutionary biologist at the AMNH used HTC services provided by the OSG to unlock a genomic basis for convergent evolution in bats.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-10-26-banq/">Centuries of newspapers are now easily searchable thanks to HTCSS</a>
          
          <small class="text-muted article-date">October 26, 2021</small>
          
        </h2>
        <p>BAnQ’s digital collections team recently used HTCSS to tackle their largest computational endeavor yet –– completing text recognition on all newspapers in their digital archives.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-19-Showcase/">Transforming research with high throughput computing</a>
          
          <small class="text-muted article-date">August 19, 2021</small>
          
        </h2>
        <p>During the OSG Virtual School Showcase, three different researchers shared how high throughput computing has made lasting impacts on their work.

</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-19-Spencer-Showcase/">Scaling virtual screening to ultra-large virtual chemical libraries</a>
          
          <small class="text-muted article-date">August 19, 2021</small>
          
        </h2>
        <p>Kicking off last week’s OSG User School Showcase, Spencer Ericksen, a researcher at the University of Wisconsin-Madison’s Carbone Cancer Center, described how high throughput computing (HTC) has made his work in early-stage drug discovery infinitely more scalable.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-19-Hannah-Showcase/">Using HTC for a simulation study on cross-validation for model evaluation in psychological science</a>
          
          <small class="text-muted article-date">August 19, 2021</small>
          
        </h2>
        <p>During the OSG School Showcase, Hannah Moshontz, a postdoctoral fellow at UW-Madison’s Department of Psychology, described her experience of using high throughput computing (HTC) for the very first time, when taking on an entirely new project within the field of psychology.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-19-Anirvan-Showcase/">Antimatter: Using HTC to study very rare processes</a>
          
          <small class="text-muted article-date">August 19, 2021</small>
          
        </h2>
        <p>Anirvan Shukla, a User School participant in 2016, spoke at this year’s Showcase about how high throughput computing has transformed his research of antimatter in the last five years.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-01-15-Campus-Cyberinfrastructure-Awardees-Contribute-to-OSG/">Campus Cyberinfrastructure (CC*) Awardees Contribute Over 349 Million Core Hours in Last 12 Months</a>
          
          <small class="text-muted article-date">August 12, 2021</small>
          
        </h2>
        <p>Campus Cyberinfrastructure Award Recipients Power the OSG and contributed over 349 million core hours to researchers using distributed high throughput computing (dHTC).
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-08-10-Science-Gateway/">OSG fuels a student-developed computing platform to advance RNA nanomachines</a>
          
          <small class="text-muted article-date">August 10, 2021</small>
          
        </h2>
        <p>How undergraduates at the University of Nebraska-Lincoln developed a science gateway that enables researchers to build RNA nanomachines for therapeutic, engineering, and basic science applications.

</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-06-10-OSPool-Hits-Record-Utilization/">OSPool Usage Hits Daily Record</a>
          
          <small class="text-muted article-date">June 10, 2021</small>
          
        </h2>
        <p>Researchers utilizing the OSPool are racking up record-breaking numbers. On June 8, the OSPool, which provides computing resources to researchers across the country, went over 1.1 million core hours –– a daily record number. To put this in perspective, one million core hours is equivalent to using 42 thousand cores in just one day. That is close to half the size of some large supercomputing centers. In short, an increasing number of researchers are utilizing the OSG to carry out an incredible amount of computing.

</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2021-01-15-Use-HTCondor-To-Transfer-Files/">How to Transfer 460 Terabytes? A File Transfer Case Study</a>
          
          <small class="text-muted article-date">January 15, 2021</small>
          
        </h2>
        <p>When Greg Daues at the National Center for Supercomputing Applications (NCSA) needed to transfer 460 Terabytes of NCSA files from the National Institute of Nuclear and Particle Physics (IN2P3) in Lyon, France to Urbana, Illinois, for a project they were working with FNAL, CC-IN2P3 and the Rubin Data Production team, he turned to the HTCondor High Throughput system, not to run computationally intensive jobs, as many do, but to manage the hundreds of thousands of I/O bound transfers.

</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2020-12-18-OSG-Training-and-Campus-Workshop/">Register Now for the February 8-9 Campus Workshop</a>
          
          <small class="text-muted article-date">December 18, 2020</small>
          
        </h2>
        <p>Save the date and register now for another Campus Workshop on distributed high-throughput computing (dHTC) February 8-9 offered by the Parternship to Advance Throughput Computing (PATh).
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2020-12-7-Submit-Locally-Run-Globally/">Submit Locally, Run Globally</a>
          
          <small class="text-muted article-date">December 07, 2020</small>
          
        </h2>
        <p>The PATh project offers technologies and services that enable researchers to harness through a single interface, and from the comfort of their “home directory”, computing capacity offered by a global and diverse collection of resources.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2020-11-18-High-Throughput-Computing-Powers-Covid-Genetic-Sequencing/">High Throughput Computing Powers Covid Genetic Sequencing</a>
          
          <small class="text-muted article-date">November 18, 2020</small>
          
        </h2>
        <p>MADISON - David O’Connor, professor at the UW School of Medicine and Public Health, reports that recent advances in genetic sequencing of viruses makes it possible to track the spread of SARS-CoV-2 throughout communities in Wisconsin.
</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/2020-10-20-path-awarded/">National Science Foundation establishes a partnership to advance throughput computing </a>
          
          <small class="text-muted article-date">October 10, 2020</small>
          
        </h2>
        <p>MADISON — Recognizing the University of Wisconsin-Madison’s leadership role in research computing, the National Science Foundation announced this month that the Madison campus will be home to a five-year, $22.5 million initiative to advance high-throughput computing.

</p>
      </div>
    </div>
  </div>
  
  <div class="col-12 col-md-6 mb-3">
    <div class="news news-summary">
      <div class="article-content">
        <h2 class="article-title">
          <a href="/news/htc-in-support-of-science/">High Throughput Computing in Support of Science</a>
          
        </h2>
        <p>The high throughput computing capabilities provided by HTCondor and the OSG Consortium’s Fabric of Services have a rich history of advancing all domains of research. From detecting gravitational waves caused by ancient black hole collisions, to hunting viral variants of COVID-19 –– browse the collection of articles below to discover just what’s possible with high throughput computing.

</p>
      </div>
    </div>
  </div>
  
</div>


  
        </div>
    </main>
</div>
<div class="footer">
  <div class="container">
    <div class="row">
      <div class="col-12">
        <div class="footer-inner">
          <h3 class="footer-title">PATh</h3>
          <ul>
            
            
            <li class="">
              <a href="/">Home</a>
            </li>
            
            <li class="">
              <a href="/contact/">Contact</a>
            </li>
            
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
<div class="sub-footer">
  <div class="container">
    <div class="row">
      <div class="col-12">
        <div class="sub-footer-inner">
          <div class="social">
              <a href="https://www.nsf.gov/oac">
                  <img height="70px" width="70px" src="/images/logo/nsf-logo.svg">
              </a>
          </div>
          <div class="copyright">
              This project is supported by National Science Foundation under Cooperative Agreement
              <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2030508">OAC-2030508</a>.
              Any opinions, findings, conclusions or recommendations expressed in this material are those of the
              authors and do not necessarily reflect the views of the National Science Foundation.
          </div>
        </div>
      </div>
    </div>
  </div>
</div>

<script type="text/javascript" src="/assets/js/scripts.js"></script>

    
    
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-0CCSDQV212"></script>
    <script>
      window.dataLayer = window.dataLayer || [];

      function gtag() {
        dataLayer.push(arguments);
      }
      gtag('js', new Date());
      gtag('config', 'G-0CCSDQV212');
    </script>
    

</body>
</html>